helm/hpcc/templates/thor.yaml

```diff
@@ -267,6 +280,10 @@ spec:
                 {{ include "hpcc.daliArg" $ }},
                 {{ printf "\"--name=%s\"" .name }}
               ]
+        env:
+{{ include "hpcc.mergeEnvironments" $env | indent 8 -}}
```

Comment:
These sentinal variables are new.  (Also localroxie).  Could it have caused problems before, or are they unused, but here for completeness?